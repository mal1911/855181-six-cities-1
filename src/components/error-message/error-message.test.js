import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

it(`ErrorMessage correctly renders`, () => {
  const tree = renderer
    .create(<ErrorMessage
      message={`Error`}
      onToggle={jest.fn()}
      opened={true}
    />).toJSON();
  expect(tree).toMatchSnapshot();
});
