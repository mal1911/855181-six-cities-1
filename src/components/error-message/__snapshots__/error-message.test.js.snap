// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage correctly renders 1`] = `
<div
  className="error-message error-message--opened"
  onClick={[Function]}
>
  <p
    className="error-message__test"
  >
    Ошибка: Error!
  </p>
</div>
`;
